example/src/modules/x/child/child.js

```diff
@@ -2,9 +2,4 @@ import { LightningElement, api } from 'lwc';
 
 export default class App extends LightningElement {
   @api message = '';
-  get badmessage() {
-    // Intentionally rendering differently in SSR & CSR to demonstrate
-    // the playground functionality.
-    return import.meta.env.SSR ? `${this.message} oops SSR oops` : this.message;
-  }
```

Comment:
`npm run test` was failing prior to this change. I'm not sure why it wasn't caught by CI before. We'll need to look into that to ensure tests are running correctly for our PRs.